airbyte-integrations/connectors/source-amplitude/source_amplitude/api.py

```diff
@@ -231,8 +244,12 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
         if response_data:
             series = list(map(list, zip(*response_data["series"])))
             for i, date in enumerate(response_data["xValues"]):
-                yield {"date": date, "statistics": dict(zip(response_data["seriesLabels"], series[i]))}
-
+                try:    
+                    yield {"date": date, "statistics": dict(zip(response_data["seriesLabels"], series[i]))}
+                except (IndexError,KeyError) as e:
+                    #To avoid propogating this error to UI
```

Comment:
What does this mean?

Reply:
List Index Out of range error is thrown by line 248 for series[i]. Handling that here. Put in an additional check to check if seriesLabels is present in response_data.